jwst/pipeline/calwebb_ami3.py

```diff
@@ -10,7 +10,7 @@
 from ..ami import ami_analyze_step
 from ..ami import ami_average_step
 from ..ami import ami_normalize_step
-
+from ..resample import blend
```

Comment:
We really should put the blend stuff in its own package, or the lib area, in the future.